svc-lighthouse-api/src/main/java/gov/va/vro/abddataaccess/service/FieldExtractor.java

```diff
@@ -32,6 +32,38 @@ public class FieldExtractor {
 
   private static final String SNOMED = "http://snomed.info";
   private static final String MISSING = "*Missing*";
+  public static final String SYSTOLOC_BP_CODE = "8480-6";
+  public static final String DIASTOLOC_BP_CODE = "8462-4";
+  private static final String BP_UNIT = "mm[Hg]";
+  private static final String SYSTOLIC_DES = "Systolic blood pressure";
+  private static final String DIASTOLIC_DES = "Diastolic blood pressure";
+
+  public enum BP_MEASURE {
```

Comment:
This is a wrong convention of enum names. We get two warnings during build as expected. Please just use BpMeasure. Also add a javadoc.